stdlib/public/core/CollectionAlgorithms.swift.gyb

```diff
@@ -211,6 +211,56 @@ extension MutableCollection where Self : BidirectionalCollection {
   }
 }
 
+//===----------------------------------------------------------------------===//
+// shuffled()
+//===----------------------------------------------------------------------===//
+
+% for Self in ['Sequence', 'MutableCollection']:
```

Comment:
We don't need a different version of `shuffled` for `MutableCollection`—just the `Sequence` version is fine. (I think you're following the example of `sorted` here, but that's actually redundant, too.)